can/bus.py

```diff
@@ -235,6 +236,10 @@ def send_periodic(
             Function which should be used to modify each message's data before
             sending. The callback modifies the :attr:`~can.Message.data` of the
             message and returns ``None``.
+        :param period_intra:
+            Period in seconds between each message when sending multiple messages
+            in a sequence. If not provided, the period will be used for each
+            message.
```

Comment:
Period in seconds between individual message within the sequence. 

I assume no impact for sending individual messages? Could you also update the docstring for `period` 🙏🏼